### ISSUE TITLE
HTML injection freaking out

### ISSUE THREAD

**Issue by OoM-JaN** [tzapu/WiFiManager#916]
Hi 

Great piece of work. It's a great library but I'm stuck in the HTML insert.
Here is my code (partially):

```
String HtmlPage(){
    String htmlPageNoWiFi =
     String ( "<H1>TEST</H1>");
    return htmlPageNoWiFi;
}

void setup(){

  String y = HtmlPage();
  const char* z = "<H1>TEST</H1>";
  WiFiManagerParameter custom_text(z);

  wifiManager.addParameter(&custom_text);
  Serial.println(y);
  Serial.println(z);
```

This works perfectly and both 'y' and 'z' give the same output on the serial monitor.
If I change   WiFiManagerParameter custom_text(z); to   WiFiManagerParameter custom_text (y); I receive an error:
> no matching function for call to 'WiFiManagerParameter::WiFiManagerParameter(String&)'

What am I doing wrong or how can I change (y) to work within the WifiManager?

**Comment by tablatronix** [comment]
convert to const char* ?

**Comment by OoM-JaN** [comment]
const char* is the only way that custom_text 'eats' the String.
`WiFiManagerParameter custom_text("<H1>TEST</H1>") `
Works as well.

```
String z = "<H1>TEST</H1>";
'WiFiManagerParameter custom_text(z);
```
Doesn't work

**Comment by tablatronix** [comment]
Yeah you are trying to pass a string in, there is no string handler, maybe we should add one

**Comment by wvsbsp** [comment]
does ths work?
String z = "<H1>TEST</H1>";
'WiFiManagerParameter custom_text(z.c_str());
